checks/fifteen.py

```diff
@@ -0,0 +1,68 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import File, Test, Error, check
+
+class Fifteen(Test):
```

Comment:
@crossroads1112 worked out a way to do this today by swapping out instances of `callq foo` in the assembly file to `callq bar`, where `foo` is the student's version of the function and `bar` is ours.